Mod2/Labs/Week3/SQLAggregates.md

```diff
@@ -0,0 +1,30 @@
+## SQL Aggregates
+
+**Data in the Real World**
+Our set list dataset is fun, and it helps us learn the mechanics of SQL.  But, SQL can be much more impactful than analyzing song data.  In fact, there is a whole field of study called `Data Analytics` that relies heavily on querying data to help us learn more about the world and our place in it.
+
+One of the first projects new data analysts work on to learn their craft is an analysis of the [Titanic](https://en.wikipedia.org/wiki/Titanic) passengers.  In our lab today, we are going to do an abbreviated version of this project.
```

Comment:
I would probably change the wording here to "A common first project for new data analysts is", just to make sure it doesn't sound like everyone analyzes this data set.